### PR TITLE
Feature/remove lights from bridge

### PR DESCRIPTION
**Pull request by jlunenburg**
* Needs testing
* Preparation (both robots)
```bash
roscd robot_skills; git pull --ff-only; git checkout feature/remove-lights-from-bridge
roscd hero_bringup; git pull --ff-only; git checkout feature/remove-lights-from-bridge
roscd hero_bridge; git pull --ff-only; git checkout feature/remove-lights-from-bridge
```
* Testing
    * In the first terminal (on hero1): `hero-start`
    * In the second terminal (on hero2): `hero-free-mode`
    * In the third terminal (on hero2): `hero-console`. This opens an iPython console
        * `robot.lights.set_color(1, 0, 0)` --> robot lights should turn red
        * `from robot_skills import lights`
        * `robot.lights.set_color_rgba_msg(lights.RESET)` --> robot lights should turn blue
* Reset (both robots)
 ```bash
roscd robot_skills; git checkout master
roscd hero_bringup; git checkout master
roscd hero_bridge; git checkout master
```
* Please add your feedback in this MR
* Merge together with https://github.com/tue-robotics/hero_bridge/pull/26 and https://github.com/tue-robotics/hero_bringup/pull/82

### CODE REVIEW

File: robot_skills/src/robot_skills/lights.py [hunk 1]
```diff
@@ -39,45 +34,58 @@ def set_color(self, r, g, b, a=1.0):
...
 
-        :return: no return
+        :param robot_name: robot_name
+        :param tf_listener: tf_server.TFClient()
         """
-        rgb_msg = RGBLightCommand(show_color=True)
-        self._topic.publish(rgb_msg)
+        super(TueLights, self).__init__(robot_name=robot_name, tf_listener=tf_listener)
+        self._publisher = rospy.Publisher(
+            '/'+robot_name+'/rgb_lights_manager/user_set_rgb_lights', RGBLightCommand, queue_size=10
```
Comment (MatthijsBurgh) [inline robot_skills/src/robot_skills/lights.py:77]: Use format instead of string appending?

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

File: robot_skills/src/robot_skills/lights.py (hunk 1 above)
Comment (jlunenburg) [reply robot_skills/src/robot_skills/lights.py:77]: I'd be happy to change it but not sure why. In any way, in Python 3 we can use f-strings

Comment (MatthijsBurgh) [reply robot_skills/src/robot_skills/lights.py:77]: Yes, I prefer it that way. Plus it is a little bit faster I think. F strings have to wait a month

Comment (jlunenburg) [reply robot_skills/src/robot_skills/lights.py]: Not sure if it's faster (see [StackOverflow](https://stackoverflow.com/questions/38722105/format-strings-vs-concatenation)) I changed it in 0773c51f963052903cd1ff4b008659b51f45d874

File: hero_skills/src/hero_skills/hero.py [hunk 2]
```diff
@@ -42,16 +42,20 @@ def __init__(self, wait_services=False):
                                                                camera_base_ns='hero/head_rgbd_sensor'))
 
         # Human Robot Interaction
-        self.add_body_part('lights', lights.Lights(self.robot_name, self.tf_buffer))
+        self.add_body_part(
+            'lights', lights.Lights(
+                self.robot_name, self.tf_buffer, '/' + self.robot_name + '/rgb_lights_manager/user_set_rgb_lights'
```
Comment (MatthijsBurgh) [inline hero_skills/src/hero_skills/hero.py:47]: HERO listens to `/hero/command_status_led_rgb`

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

**Comment by MatthijsBurgh** [comment]
As the diagnostics also publish on this topic, it can become a bit of a mess/disco. Also the default blue is much darker than the blue from the diagnostics node.

File: robot_skills/src/robot_skills/lights.py [hunk 3]
```diff
@@ -39,45 +34,58 @@ def set_color(self, r, g, b, a=1.0):
...
+
+class TueLights(LightsInterface):
+    def __init__(self, robot_name, tf_buffer):
         """
-        Set the lights of the robot ON
+        Interface to the robot's lights. This uses the TU/e-specific RGBLightCommand message type.
 
-        :return: no return
+        :param robot_name: robot_name
+        :param tf_buffer: tf_server.TFClient()
```
Comment (MatthijsBurgh) [inline robot_skills/src/robot_skills/lights.py:73]:
```suggestion
        :param tf_buffer: tf buffer object
```

File: robot_skills/src/robot_skills/lights.py [hunk 4]
```diff
@@ -118,3 +126,24 @@ def blue(t):
...
             rate.sleep()
+
+
+class Lights(LightsInterface):
+    def __init__(self, robot_name, tf_buffer, topic):
+        """
+        Interface to the robot's lights. This uses the TU/e-specific RGBLightCommand message type.
+
+        :param robot_name: robot_name
+        :param tf_buffer: tf_server.TFClient()
```
Comment (MatthijsBurgh) [inline robot_skills/src/robot_skills/lights.py:137]:
```suggestion
        :param tf_buffer: tf buffer object
```

File: hero_skills/src/hero_skills/hero.py (hunk 2 above)
Comment (MatthijsBurgh) [reply hero_skills/src/hero_skills/hero.py:47]:
```suggestion
                self.robot_name, self.tf_buffer, '/hero/command_status_led_rgb'
```

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**